src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
```

Comment:
According to the checklist, don't begin class implementation with an empty line. Remove the empty line before the class declaration.